scripts/cluster_settings.ovn

```diff
@@ -0,0 +1,6 @@
+# Specific settings for the submariner E2E
+cluster_nodes['cluster1']="control-plane worker"
+cluster_nodes['cluster2']="control-plane worker worker"
+cluster_nodes['cluster3']="control-plane worker worker"
```

Comment:
Can we include _cluster_settings_ as a base so we don't duplicate this?

Reply:
Actually we can remove this coz ovn kind script ignores this.